src/nuclio/functions/version/version-configuration/tabs/version-configuration-labels/version-configuration-labels.component.js

```diff
@@ -86,6 +86,7 @@
                 })
                 .value();
             ctrl.labels = lodash.compact(ctrl.labels);
+            ctrl.addNewLabelTooltip = ctrl.isVersionDeployed(ctrl.version) ? $i18next.t('functions:TOOLTIP.ADD_LABELS', {lng: lng}) : '';
```

Comment:
The line should have no more 120 characters

Reply:
rename `addNewLabelTooltip`. Because the tooltip not only for adding the new Label